src/battleground/HealthBarsField.js

```diff
@@ -0,0 +1,32 @@
+//@flow strict
+
+import * as React from 'react';
+import "../Main.css";
```

Comment:
import Main.css is unnecessary. As long as it's imported on any of our components, it will extend to the whole website.